codex-ui/src/vue/components/alert/BaseAlert.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div
+    ref="el"
+  >
+    <div
+      :class="[
+        $style.alert,
+        $style['alert--' + computedStyle]
+      ]"
+    >
+      <Icon
+        v-if="icon"
+        :name="icon ?? null"
+      />
+      <div>{{ props.message }}</div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, computed, withDefaults, ref } from 'vue';
+import Icon from '../icon/Icon.vue';
+import type { AlertOptions, AlertType } from './core/types';
+import { ALERT_CONTAINER_STYLES } from './core/constant';
+
+const el = ref<HTMLElement>();
+
+const props = withDefaults(defineProps<AlertOptions>(), {
+  id: ALERT_CONTAINER_STYLES.id,
+  position: ALERT_CONTAINER_STYLES.position,
+  message: ALERT_CONTAINER_STYLES.message,
+  icon: ALERT_CONTAINER_STYLES.icon,
+  type: ALERT_CONTAINER_STYLES.type,
+  timeout: ALERT_CONTAINER_STYLES.timeout,
+});
+
+/**
+ * compute serveral style type
+ */
+const computedStyle = computed<AlertType>(() => {
+  if (props.type === 'success') {
+    return 'success';
+  }
+
+  if (props.type === 'error') {
+    return 'error';
+  }
+
+  if (props.type === 'warning') {
+    return 'warning';
+  }
+
+  if (props.type === 'info') {
+    return 'info';
+  }
+
+  return 'default';
+});
+
+</script>
+
+<style module lang="postcss">
+.alert {
+  --padding-left: var(--h-padding);
+  --padding-right: var(--h-padding);
+  --color: var(--accent--text-solid-foreground);
+  --bg: var(--base-text);
```

Comment:
bg is `solid` in figma (except "Default" where bg is `bg-secondary`)

Reply:
actual

Reply:
added